dev/src/collection-group.ts

```diff
@@ -168,18 +172,23 @@ export class CollectionGroup<T = firestore.DocumentData>
    *   post.someNonExistentProperty; // TS error
    * }
    *
-   * @param {FirestoreDataConverter} converter Converts objects to and from
-   * Firestore.
+   * @param {FirestoreDataConverter | null} converter Converts objects to and
```

Comment:
ultranit: The "C" in "Converts" should be lowercase. Also in `reference.ts`.

Reply:
Thanks for catching!

Reply:
Actually, it should be capitalized, since it's the description of the `converter` parameter. The other docs descriptions capitalize the first letter as well.

Reply:
Ohhhh I'm a dummy. Sorry for the false alarm!